keps/sig-api-machinery/1040-priority-and-fairness/README.md

```diff
@@ -847,69 +847,156 @@ Because we now have more possible values for `mu(i,t)` than 0 and
 `mu_fair(t)`, it is more computationally complex to adjust the
 `mu(i,t)` values when a packet arrives or completes virtual service.
 That complexity is:
-- O(n log n), where n is the number of queues,
+- `O(n log n)`, where n is the number of queues,
```

Comment:
I guess it might be obvious to you, but I think I'm not 100% how exactly we're computing those values here (I know it's technically not related to this PR, but to earlier changes, but it would be good to describe it in a bit more details).

Reply:
https://github.com/kubernetes/kubernetes/blob/f79795d71850ab0054b0759f77dd0cec17bb4213/staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset_test.go#L41-L74

@MikeSpreitzer ^^ the code solves max-min fairness in a time complexity of `O(nlogn + 2n)`, is it?

Reply:
btw in theory, i think the time complexity can be optimized to `O(n)` by leveraging some efficient linear-programming resolving algorithms

Reply:
I would be surprised at that, because the '<' relationships among the 'rho' values are part of the definition of the problem and sorting requires order `n log n`.

Reply:
The current text is good enough - thanks.